client/signup/steps/plans/index.jsx

```diff
@@ -250,17 +252,18 @@ PlansStep.propTypes = {
 };
 
 const PlansStepWithMonthlyPricingTest = ( props ) => {
-	const isEligibleForMonthlyPricing = 'onboarding' === props.flowName;
+	const isLoading = useIsLoading();
+	const variation = useVariationForUser( 'monthly_pricing_test_phase_1' );
+	const isEligibleForMonthlyPricing =
+		'onboarding' === props.flowName && 'treatment' === variation && ! isIgnoredAdSource();
 
 	return (
-		<Experiment name="monthly_pricing_test_phase_1">
-			<DefaultVariation name="control">
-				<PlansStep { ...props } isMonthlyPricingTest={ false } />
-			</DefaultVariation>
-			<Variation name="treatment">
+		<>
+			<QueryExperiments />
+			{ ! isLoading && (
 				<PlansStep { ...props } isMonthlyPricingTest={ isEligibleForMonthlyPricing } />
```

Comment:
I forgot to mention, it may be a good idea to add a prop to a tracks event (or fire a new one) that includes the status of `isEligibleForMonthlyPricing` which will help filter those out if this does end up biasing the test in some way. It's not a blocker, just a heads up; just in case.

Reply:
I like this suggestion, just slightly altered!

Consider a separate variable `isEligibleForMonthlyPricingExperiment` which checks for 
```
'onboarding' === props.flowName && ! isIgnoredAdSource()
```

And when true, then trigger a Tracks event like `calypso_experiment_exposure_eligible` with eventprop `experiment_name: monthly_pricing_test_phase_1`. You can then use that as your exposure event in your Abacus experiment analysis.

This will help ensure you are only analyzing over users that are eligible for your experiment. Without this particular exposure event, you may be diluting the experiment analysis by preventing some `treatment` participants from experiencing the monthly plans pricing.